tests/unit/remote/test_local.py

```diff
@@ -0,0 +1,42 @@
+from dvc.repo import Repo
+from dvc.remote.local import RemoteLOCAL
+
+
+def test_status_download_optimization(mocker):
+    """When comparing the status to pull a remote cache,
+        And the desired files to fetch are already on the local cache,
+        Don't check the existance of the desired files on the remote cache
+    """
+    remote = RemoteLOCAL(Repo(), {})
```

Comment:
You are passing dvc repo instance of the dvc project itself. It is not right. Didn't `RemoteLOCAL(None, {})` work?

Reply:
Sorry! `None` intended :disappointed:

Reply:
Ready, @efiop